docs/user/plot_radar_fov.md

```diff
@@ -0,0 +1,70 @@
+# Fan plots
+---
+
+Fan plots are a way to visualise data from the entire scan of SuperDARN radar. It is when the data in all beams and ranges for a given parameter (such as line-of-sight velocity, backscatter power, etc) and a particular scan is projected onto a magnetic local time/magnetic latitude plot in [AACGMv2 coordinates](http://superdarn.thayer.dartmouth.edu/aacgm.html).
+
+Currently, fan plots in pyDARN get the geographic positions of a radars range gates automatically by reading in pre-generated files (found in the `/pydarn/radar_fov_files` folder), and then [converts](https://pypi.org/project/aacgmv2/) them to AACGMv2 coordinates. This will change in the future, such that pyDARN will work out the positions itself. That will bring suport for not standard range/beam layouts.
+
+### Basic usage
+pyDARN and pyplot need to be imported, as well as any FITACF file needs to be [read in](https://pydarn.readthedocs.io/en/master/user/SDarnRead/):
+
+```python
+import matplotlib.pyplot as plt
+import pydarn
+
+#Read in fitACF file using SDarn_read
+file = "path/to/fitacf/file"
+SDarn_read = pydarn.SDarnRead(file)
+fitacf_data = SDarn_read.read_fitacf()
+
+```
+With the FITACF data loaded as a list of dictionaries (`fitacf_data` variable in above example), you may now call the `plot_radar_fov` method. Make sure you tell it which scan (numbered from first recorded scan in file) you want using `scan_index`:
+```python
+a = pydarn.fan.plot_radar_fov(fitacf_data, scan_index=27)
+plt.show()
+
+```
+Here I plotted the 27th scan in the file, and because I didn't tell it which parameter to use, it has defaulted to line-of-sight velocity:
+![](../imgs/fan_1.png)
+
+Default plots also do not show groundscatter as grey. Set it to true to colour groundscatter this way:
+```python
+a = pydarn.fan.plot_radar_fov(fitacf_data, scan_index=27, groundscatter=1)
```

Comment:
I know I mentioned this earlier, and Dan said it would be in a future release. But to document that discussion start/end time rather than scan_index might be more intuitive. 
(to grab the nearest scan to time or something, but also plotting partial scans would be cool, with something Devin was talking about doing)